docs/src/reference/core/changelog.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+0.2.2 (UNRELEASED)
+------------------
+
+Fixed
+*****
+
+* fix preprocessing of config io fields
```

Comment:
Mind giving more of an explanation? It can still be a single sentence - just something that makes more sense for the user (i.e. include what was the problem)